### PR TITLE
2-15-2018 4 Commits: USER_ID / Remove Icons Dir (+1) AND 1-27-2018 DELETE AND INLINE EDIT +3

### PR DESCRIPTION
**Pull request by iremaka**
2-15-2018 -> Show user ID (non editable) and removed unused "icons" directory

ALSO - > Implemented the DELETE feature. Turned "Actions" into a blue button for
each row that opens a context menu on left clicking the button -
currently there is one option for each row: "Delete." It asks "Are you
sure" after clicking "Delete" - if you say yes, it will delete the row
immediately, and it also deletes the correct tree from the database.
------------------
This is the new INLINE EDIT feature for the admin panel.

note that public\datatables.min.js also includes
jQuery 3 3.2.1, DataTables 1.10.16, Buttons 1.5.1
and also
CellEdit 1.0.19 (modded by iremaka)

CellEdit (which is modded from the original version, primarily to be
able to handle number input fields and moreover to handle them with a
'step' parameter) is the main utility by which inline edit of the tables
is being acccomplished

Dashboard.js and Record.js were modified to add a few more columns:
namely Id, Priority (to test usage of boolean columns), and also, lat
and long were split into distinct columns.

c1.js is custom code and is the main implementation of the new inline
edit feature.

NOTE: in general, within App.js.,
better not to use componentWillMount in general, including for fetching
data
instead prefer to use componentDidMount()

### CODE REVIEW

File: public/c1.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
+//created 1-27-2018 iremaka
+ 
+    var table7 = $('#table7').DataTable();
+
+
+
+    function table7cb(updatedCell, updatedRow, oldValue) {
+        
+        //console.log("The new value for the cell is: " + updatedCell.data());
```
Comment (shakyink) [inline public/c1.js]: We could probably remove the console.log comments if there not being used anymore.

**Comment by shakyink** [comment]
@iremaka - Couldn't we move the logic  and dependencies in `public` for the delete and edit under the `src` where we can import the dependencies using `import` from the `node_modules`?

- `src` directory serves as an entry point to the Javascript. 
- `public` is only used for the page template for the components.

**Comment by iremaka** [comment]
2-15-2018 - show user_id (non editable) and also removed icons directory (unused)

**Comment by deepwinter** [comment]
Thanks for this PR !

**Pull request merged**